src/main/kotlin/stringAddCalculator/StringAddCalculator.kt

```diff
@@ -0,0 +1,44 @@
+package stringAddCalculator
+
+object StringAddCalculator {
+    private const val CUSTOM_DELIMITER =
+        """//(.)\\n(.*)"""
+    private const val ZERO = 0
+    private const val NEGATIVE_NUMBER_THROW_MESSAGE = "can not use nagative number"
+    private val delimiters = mutableListOf(",", ":")
+
+    fun add(text: String): Int {
+        if (text.isBlank()) return 0
+        if (text.length == 1) return text.toInt()
+        var targetText = getTargetText(text)
+        val numbers = getNumbers(targetText, delimiters)
```

Comment:
`delimiters`가 꼭 필요할까요?

```suggestion
        val numbers = getNumbers(targetText)
```